src/apps/bristleback_apps/bm_rbr/user_code/rbr_sensor.cpp

```diff
@@ -108,10 +116,10 @@ bool RbrSensor::probeType(uint32_t timeout_ms) {
       printf("RBR sensor was lost\n");
     }
   } else {
-    if (_sensorDropDebounceCount) {
-      bm_fprintf(0, RBR_RAW_LOG, "RBR sensor was recovered\n");
-      bm_printf(0, "RBR sensor was recovered\n");
-      printf("RBR sensor was recovered\n");
+    if (_sensorDropDebounceCount >= SENSOR_DROP_DEBOUNCE_MAX_COUNT) {
+      bm_fprintf(0, RBR_RAW_LOG, "RBR sensor online\n");
+      bm_printf(0, "RBR sensor online\n");
+      printf("RBR sensor online\n");
```

Comment:
"recovered" kind of implied the mote took action to put the sensor back online. 

We have a different FTL print for this and the online detection should be kept distinct.